lib/iris/tests/experimental/regrid/test_regrid_area_weighted_rectilinear_src_and_grid.py

```diff
@@ -495,39 +466,35 @@ def test_scalar_source_cube(self):
         self.assertEqual(res.data, src.data)
 
     @tests.skip_data
-    @tests.skip_plot
     def test_global_data_reduce_res(self):
         src = iris.tests.stock.global_pp()
         src.coord('latitude').guess_bounds()
         src.coord('longitude').guess_bounds()
         dest = _resampled_grid(src, 0.4, 0.3)
         res = regrid_area_weighted(src, dest)
-        qplt.pcolormesh(res)
-        self.check_graphic()
+        self.assertArrayAlmostEqual(res.data.mean(), 280.137698)
```

Comment:
So it's a precision issue? In that case, maybe a hash of data would be better to test. NumPy arrays can't be hashed, but the underyling buffer (`ndarray.data`) can be, if the `writeable` flag of the `ndarray` is  set to `False`.

I'm wondering if testing the mean/standard deviation means we wouldn't catch bugs which only changed the data slightly. It's probably nothing to worry about, but just a thought.